common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -126,6 +126,15 @@ public class LootrunModel extends Model {
 
     private Map<BeaconColor, Integer> selectedBeacons = new TreeMap<>();
 
+    private int timerOverall = 0;
+
+    @Persisted
+    private Storage<Map<String, Integer>> timerChallenge = new Storage<>(new TreeMap<>());
+
+    ;
```

Comment:
oops

Reply:
I think the IntelliJ's Git integration was messing with my lines and I got lost and this came out by an accident, heh